### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by merwok**
This time I wasn’t lazy and tested with django 1.8 to 2.0.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 language: python
-python: 3.5  # this is needed to get travis to have python3.5 as well
+python: 3.6  # this is needed to get travis to have python3.6 as well
```
Comment (merwok) [inline .travis.yml]: Note that I don’t use TravisCI so I don’t know if that setting is valid.  I assume it is!

File: django_webtest_tests/settings.py [hunk 2]
```diff
@@ -8,71 +8,42 @@
...
 
-
-DEBUG = True
-
-ADMINS = (
-    # ('Your Name', 'your_email@domain.com'),
-)
-
-MANAGERS = ADMINS
-
```
Comment (merwok) [inline django_webtest_tests/settings.py]: I just removed useless lines and comments here.  I can drop that commit if you’d prefer.

File: django_webtest_tests/settings.py [hunk 3]
```diff
@@ -89,7 +60,8 @@
     },
 ]
 
-if django.VERSION < (1, 8):
+
+if django.VERSION < (1, 10):
```
Comment (merwok) [inline django_webtest_tests/settings.py]: This is the bug in my previous PR.

File: tox.ini [hunk 4]
```diff
@@ -22,7 +23,6 @@ whitelist_externals = env
 changedir=django_webtest_tests
 commands=
     env
-    pip freeze
```
Comment (merwok) [inline tox.ini]: tox already prints installed packages.

**Comment by gawel** [comment]
It still broken because travis no longer provide all python versions in containers

**Comment by gawel** [comment]
fixed using a matrix

**Pull request merged**